--- vgmstream/src/CMakeLists.txt	2025-01-01 01:18:07.000000000 +0800
+++ vgmstream/src/CMakeLists.txt	2025-05-03 10:47:33.285600800 +0800
@@ -56,6 +56,17 @@
 	layout
 	meta
 	util
+	${CMAKE_CURRENT_SOURCE_DIR}
+	${CMAKE_CURRENT_SOURCE_DIR}/base
+	${CMAKE_CURRENT_SOURCE_DIR}/coding
+	${CMAKE_CURRENT_SOURCE_DIR}/layout
+	${CMAKE_CURRENT_SOURCE_DIR}/meta
+	${CMAKE_CURRENT_SOURCE_DIR}/util
+	${VGM_SOURCE_DIR}/src/base
+	${VGM_SOURCE_DIR}/src/coding
+	${VGM_SOURCE_DIR}/src/layout
+	${VGM_SOURCE_DIR}/src/meta
+	${VGM_SOURCE_DIR}/src/util
 )
 
 # libvgmstream.a
