--- vgmstream/cmake/dependencies/ffmpeg.cmake	2025-05-02 19:49:06.518143000 +0800
+++ vgmstream/cmake/dependencies/ffmpeg.cmake	2025-05-02 19:57:41.623002300 +0800
@@ -113,6 +113,14 @@
 				--disable-nvenc
 				--disable-nvdec
 				--disable-vdpau
+				--target-os=android
+				--enable-cross-compile
+				--cross-prefix=${FFMPEG_CROSS_PREFIX}-
+				--arch=${FFMPEG_ARCH}
+				--strip=${FFMPEG_STRIP}
+				--sysroot=${FFMPEG_SYSROOT}
+				--enable-neon
+				--prefix=${FFMPEG_PREFIX}
 				--disable-vulkan
 				--enable-parser=${FFMPEG_CONF_PARSER}
 				--enable-demuxer=${FFMPEG_CONF_DEMUXER}
