--- vgmstream/cmake/dependencies/ffmpeg.cmake	2025-01-01 01:18:07.000000000 +0800
+++ vgmstream/cmake/dependencies/ffmpeg.cmake	2025-05-03 09:01:34.385108900 +0800
@@ -61,8 +61,8 @@
 
 		FetchDependency(FFMPEG
 			DIR ffmpeg
-			GIT_REPOSITORY https://git.ffmpeg.org/ffmpeg.git
-			GIT_TAG n7.1.1
+			GIT_REPOSITORY https://github.com/FFmpeg/FFmpeg.git
+			GIT_TAG n5.1.2
 		)
 
 		if(FFMPEG_PATH)
@@ -113,6 +113,16 @@
 				--disable-nvenc
 				--disable-nvdec
 				--disable-vdpau
+				--target-os=android
+				--enable-cross-compile
+				--arch=${FFMPEG_ARCH}
+				--strip=${FFMPEG_STRIP}
+				--sysroot=${FFMPEG_SYSROOT}
+				--cc=${FF_CC}
+				--cxx=${FF_CXX}
+				--ar=${FF_AR}
+				--ranlib=${FF_RANLIB}
+				--enable-neon
 				--disable-vulkan
 				--disable-libdrm
 				--enable-parser=${FFMPEG_CONF_PARSER}
